### PR TITLE
SIM/DIGI/RECO changes for HF QIE10 (part B)

### PR DESCRIPTION
**Pull request by kpedro88**
This is the second part of #13232, which was split into 2 PRs. For the first part, see #13331.

@ggovi @franzoni - please continue the review here if you have further comments.
@civanch, your review would be appreciated also.
@slava77, this is identical to what you've already approved.

I rebased this branch up to the HEAD of the CMSSW_8_0_X branch, which includes #13331, so it should be ready to test.

### CODE REVIEW

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by kpedro88** [comment]
Forgot to git add a few files, fixed now. Is this still targeted for 80X or should we move to the auto-generated #13344?

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by kpedro88** [comment]
No changes to existing workflows observed in the comparison, as expected.

@ggovi - please let me know if you are satisfied with the HcalQIEType enum in its current location. I can move it if another more appropriate and sufficiently general location is suggested.

**Comment by franzoni** [comment]
+1

thanks for https://github.com/cms-sw/cmssw/pull/13232#discussion_r53189036 , 
also supported by the fact that the 2015(D) relvals don't show changes.
cheers, @mmusich @franzoni

**Comment by franzoni** [comment]
checks failing:
- two fast sim in relvals
- 2 premix fulsim in add-ons

File: CalibCalorimetry/HcalPlugins/src/HcalHardcodeCalibrations.cc [hunk 1]
```diff
@@ -38,10 +36,10 @@ namespace {
     int maxDepthHB=hcaltopology.maxDepthHB();
     int maxDepthHE=hcaltopology.maxDepthHE();
 
-#ifdef DebugLog
-    std::cout << std::endl << "HcalHardcodeCalibrations:   maxDepthHB, maxDepthHE = " 
-	      <<  maxDepthHB << ", " <<  maxDepthHE << std::endl;
-#endif
+  /*
```
Comment (slava77) [inline CalibCalorimetry/HcalPlugins/src/HcalHardcodeCalibrations.cc]: this isn't in #13232 and is also not really the way it should show up (commented out "cout"s)

File: Configuration/EventContent/python/EventContent_cff.py [hunk 2]
```diff
@@ -303,7 +303,7 @@
 FEVTDEBUGHLTEventContent = cms.PSet(
     outputCommands = cms.untracked.vstring('drop *'),
     splitLevel = cms.untracked.int32(0),
-    eventAutoFlushCompressedSize=cms.untracked.int32(10*1024*1024)
+    eventAutoFlushCompressedSize=cms.untracked.int32(1*1024*1024)
```
Comment (slava77) [inline Configuration/EventContent/python/EventContent_cff.py]: was this PR appropriately (re)based?
This change for 10\* to 1\* is undoing the version in 80X.

**Comment by ggovi** [comment]
@kpedro88 
My recommendation was in the view of future changes. There is currently no data written with the HcalQIEType CondFormat. Therefore, it is ok provided you won't change the class in the future.

**Comment by kpedro88** [comment]
@slava77 - I addressed the rebase issue and the problems you noticed have been fixed now.

@ggovi - Okay. Well, it seems plausible we might want to extend the enum at some point (in case of different hardware versions of QIE10/QIE11), so maybe it should be moved. Perhaps a small header could be added to DataFormats/HcalDigi?

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by kpedro88** [comment]
@ggovi - I moved the enum to DataFormats/HcalDigi. Please let me know if this is acceptable. Hopefully after the tests are rerun, everyone can sign off...

**Comment by mmusich** [comment]
please test

**Comment by kpedro88** [comment]
Can someone restart the tests? @mmusich @slava77 @bsunanada

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by davidlange6** [comment]
@kpedro88 - the 81x version of this PR does not merge. Could you have a look at it?

**Comment by kpedro88** [comment]
@davidlange6 - the conflict is in Eras.py. I'm not sure of the correct procedure here - if I rebase this branch to the head of CMSSW_8_1_X, it will pollute the history of the 8_0_X PR. (If you merge this one first, then I can rebase and it will only affect the 8_1_X PR...)

**Comment by kpedro88** [comment]
@civanch - can you also sign this one?

**Comment by civanch** [comment]
+1

**Comment by kpedro88** [comment]
@mmusich @franzoni @slava77 - Please sign if the PR is acceptable...

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/13334	 abaf9a4
- changes here appear to be essentially the same as in the https://github.com/cms-sw/cmssw/pull/13232 1b8d4b9, previously signed by reco https://github.com/cms-sw/cmssw/pull/13232#issuecomment-184991708. 2016 era modifications added in this PR affect only the simulation part.
- jenkins tests pass and comparisons with baseline show no differences
    - the statement above ignores outputs of 25202, which had a different set of input minbias files in step2 (\*00000/1E73E3F5-407C-E511-81C2-0025905B85B2.root file was present only in this PR test)

**Comment by kpedro88** [comment]
@mmusich @franzoni - Gentle reminder...

**Comment by mmusich** [comment]
+1

**Comment by kpedro88** [comment]
@davidlange6 - can this be merged? I'll address the merge conflict in #13344 afterward.

**Comment by kpedro88** [comment]
@davidlange6 - I'm just going to keep pinging you...

**Comment by davidlange6** [comment]
good - distracted with shorter term issues this week..

**Pull request merged**